src/components/Main/Results/ResultsCard.tsx

```diff
@@ -106,7 +107,7 @@ function ResultsCardFunction({
             {t('Results')}
           </h2>
         }
-        help={t('This section contains simulation results')}
+        help={help(t('Results'), t('This section contains simulation results'))}
```

Comment:
This pattern is repeated throughout all elements that have a 'help' attribute. The attribute receives a HelpParams object, created by the `help(title, text)` helper.